pom.xml

```diff
@@ -814,7 +812,17 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <configuration>
             <forkCount>${surefire.forkCount}</forkCount>
+            <reuseForks>true</reuseForks>
```

Comment:
Setting this here makes it impossible for the user to override it with `-DreuseForks=false` on the command line if they need to. Did you consider creating a `surefire.reuseForks` and `failsafe.reuseForks`  and a `reuseForks` profile to set them both, like what I did with `forkCount`? See #1756

Reply:
Good idea, I will implemen that.

Reply:
My latest commit should have these changes.